conf/defaults.ini

```diff
@@ -1746,3 +1746,8 @@ hidden_toggles =
 
 # Disables updating specific feature toggles in the feature management page
 read_only_toggles =
+
+#################################### Public Dashboards #####################################
+[public_dashboards]
```

Comment:
Should this be a field of the `[dashboards]` section instead of a section? I don't have a strong opinion, just wondering  😅

Reply:
I don't have a strong opinion but I find it a bit more searchable this way. And it would be better if we have more fields regarding public dashboards (but we may never have).